ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
+    whisk_api_host={{ whisk_api_host_name | groups['edge'] | first }}
```

Comment:
@rabbah we need both, `api_host` is what we use at installing of the action, and `whisk_api_host_name` will be hostname for the callback for github webhook to call back to whisk. this will help fix https://github.com/apache/incubator-openwhisk-catalog/pull/230